data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/SqsOptions.java

```diff
@@ -43,6 +46,17 @@ public class SqsOptions {
     @DurationMin(seconds = 0)
     private Duration pollDelay = DEFAULT_POLL_DELAY_SECONDS;
 
+    @JsonProperty("max_backoff")
+    @DurationMin(seconds = 0)
+    private Duration maxBackOff = DEFAULT_MAX_BACKOFF;
+
+    @JsonProperty("base_delay")
+    @DurationMin(seconds = 0)
+    private Duration baseDelay = DEFAULT_BASE_DELAY;
+
+    @JsonProperty("max_retries")
```

Comment:
We might want to re-word this setting. It isn't exactly a maximum number of retries because this SQS ReceiveMessage is made in a loop. So this is the maximum number of retries before "failing" and starting the cycle over again.